cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1727,7 +1733,11 @@ def restore_child(self, data, **kwargs):
 class DirectoryServiceSchema(BaseSchema):
     """Represent the schema of the DirectoryService."""
 
-    domain_name = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
+    domain_name = fields.Str(
```

Comment:
same here, we can do a better validation for this string. But I think we can leave this out of this PR and track in a different task

Reply:
Removed